source/index.html.md.erb

```diff
@@ -1,23 +1,29 @@
+---
+title: Introduction
+weight: 05
+---
+
 # Introduction
 
-This guide contains the resources and documentation used by GDS for the **Getting started with the GOV.UK Prototype Kit** training.
+This guide contains the resources and documentation used by the Government Digital Service (GDS) for the 'Getting started with the GOV.UK Prototype Kit' training.
 
 It contains:
 
-+ links to the GOV.UK Protoype Kit online tutorial
++ links to the GOV.UK Prototype Kit online tutorial
 + the slides used by the facilitators to run training
-+ resources and cheatsheets on some of the concepts covered in training.
++ resources and cheatsheets on some of the concepts covered in training
 
-This training takes place across an entire day (usually from 10am – 4pm), with two short breaks in the morning and afternoon, and a lunch break.
+This training takes place across an entire day (usually from 10am to 4pm), with two short breaks in the morning and afternoon, and a lunch break.
```

Comment:
Should the 'two' be 2 here? (and the line below this)

Reply:
Good point - fixed!